src/frontend/src/views/Privacy/index.tsx

```diff
@@ -31,17 +31,17 @@ const PrivacyPolicy = (): JSX.Element => {
         <H4>Last Updated: May 17, 2021. </H4>
```

Comment:
Probably an unnecessary reminder, but we'll need to remember to update this date when we publish.